assets/js/lightbox.js

```diff
@@ -0,0 +1,7 @@
+import GLightbox from 'glightbox';
+import '../scss/global.scss'
```

Comment:
Did you mean `@import 'glightbox/dist/css/glightbox.css';` ?

Reply:
I fixed it and also some dark mode issues with the lightbox. Would you be so kind and review/merge it? :)